lib/rewrite.js

```diff
@@ -293,15 +288,12 @@ exports.search = function(js, searchRule) {
   // remove if one exists as the first line
   js = helpers.removeShebang(js);
 
-  var pattern = unwrapNode(esprima.parse(searchRule, {
+  var pattern = unwrapNode(parser.parse(searchRule, {
     raw: true
   }));
 
   var matches = [];
-  falafel(js, {
-    raw: true,
-    loc: true
```

Comment:
Well, we don't necessarily have to add it back in. It might be showing us where we are missing test coverage (assuming that it is needed). I can help run some manual tests to make sure it doesn't cause any issues.